app/controllers/posts_controller.rb

```diff
@@ -25,4 +25,12 @@ def create
       render 'new'
     end
   end
+
+  def destroy
+    @post = Post.find(params[:id])
+    authorize! :destroy, @post
```

Comment:
[Optional]
- [ ] Instead of calling authorize method for each action, you can simply use `load_and_authorize_resource` method on top of your controller to enable authorization

Reply:
@sinansevgi  Thank you so much. We used the method you told us 👍